server/application.py

```diff
@@ -119,15 +118,17 @@ def reset_jobs():
     completed = queue.Queue(maxsize=20)  # a queue of recently completed jobs
 
 
-def check_ip(ip):
-    print(ip)
-    return ip in application.config["PI_IPS"]
+def check_password(pword):
+    if pword not in application.config["FLASK_PASS"]:
```

Comment:
This looks like it's expecting multiple passwords.

Reply:
Resolved